safe-global/web-core
# fix: show newest conflict tx on dashboard

## What it solves

Resolves #669

## How this PR fixes it

The latest `timestamp`ed transaction is picked from conflict groups.

## How to test it

1. Queue a transaction.
2. Observe the queued transaction in the pending list.
3. Queue a transaction rejection.
4. Observe the transaction rejection replaces the transaction in the pending list.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/192808448-aeb655df-21aa-4d3d-b8b4-9ac0c0bbe9ae.png)

![image](https://user-images.githubusercontent.com/20442784/192808480-c5abbce6-5791-4d72-afab-8a00c7d9e2f1.png)

![image](https://user-images.githubusercontent.com/20442784/192808645-7525d7be-b3dd-4368-809f-36504e0c8e9d.png)

![image](https://user-images.githubusercontent.com/20442784/192808679-614c5379-c98d-4580-abb4-5a7c71284384.png)
